cli/src/cmd_net.rs

```diff
@@ -109,8 +109,21 @@ pub struct CmdLinkAdd {
 #[async_trait]
 impl AuthenticatedCmd for CmdLinkAdd {
     async fn run(&self, client: &Client) -> Result<()> {
-        let mut settings =
-            current_port_settings(client, &self.rack, &self.switch, &self.port).await?;
+        let mut settings = current_port_settings(client, &self.rack, &self.switch, &self.port)
+            .await
+            .unwrap_or(SwitchPortSettingsCreate {
+                addresses: HashMap::new(),
+                bgp_peers: HashMap::new(),
+                description: String::new(),
+                groups: Vec::new(),
+                interfaces: HashMap::new(),
+                links: HashMap::new(),
```

Comment:
I find that `Default::default()` is more robust, but obviously this is all good too--just sayin'!